### DISCUSSION TITLE
Sudden problem with image.

### DISCUSSION THREAD

**Discussion by LsrSal** [roleoroleo/sonoff-hack#177] [Q&A]
I have camera running remotely for a few years on 0.1.2 firmware. Couple months ago image changed and looks like 4-bit, like lower 4 bits are missing, both - color and b/w. I believe at first that after reset colors usually returning to normal for a while. No more. Now image looks like cartoon all the time. Otherwise everything appeared to work just fine. Sometimes very brightly lit portion of the image appeared with fine colors while same time other portion still looks like cartoon.
Video and snapshot looks the same.

I just upgraded to 0.1.7 - no difference.

Is there any quality settings or adjustments for image? what else can cause reduced color depth?
![image](https://github.com/user-attachments/assets/0e3d83ac-78ea-4759-9b80-84ae3112e72f)

![image](https://github.com/user-attachments/assets/c1ff9a5a-7174-4301-9e20-1bfa52e4b090)

**Comment by roleoroleo** [comment]
I don't know exactly the reason for this problem.
But it seems a hardware issue.
